src/autolabel/transforms/base.py

```diff
@@ -71,7 +81,14 @@ async def apply(self, row: Dict[str, Any]) -> Dict[str, Any]:
                 # Cache hit
                 return output
 
-        output = await self._apply(row)
+        try:
+            output = await self._apply(row)
+            output[self.transform_status_column] = True
+        except Exception as e:
+            logger.error(f"Error applying transform {self.name()}. Exception: {e}")
```

Comment:
for errors consider if error codes make sense, to keep this aligned with how errors are handled for generations